ecpy_pulses/configs/base_config.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2016 by EcpyPulses Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""
+
+"""
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
+
+
+from atom.api import (Atom, Unicode, Bool, Subclass, ForwardTyped, Typed, Dict)
+from inspect import getdoc, cleandoc
```

Comment:
Imports should be in the order specified in the ecpy documentation.

- stdlib imports
- thrid party imports
- relative imports